result/flow.yaml

```diff
@@ -113,6 +116,18 @@ components:
           description: >-
             The current rate of valid frames received
           type: number
+        min_latency:
+          description: >-
+            The minimum latency of valid frames received
+          type: integer
+        max_latency:
+          description: >-
+            The maximum latency of valid frames received
+          type: integer
+        avg_latency:
+          description: >-
+            The avarage latency of valid frames received
+          type: number
         loss:
```

Comment:
We need to specify unit of latency measurements (ns).
@ajbalogh should we include this in attr name `min_latency_ns` ? or just mention the units in description ?

Reply:
Including units would keep it consistent with bytes_tx, frames_tx etc.

Reply:
Done